telegraf/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM telegraf:latest
```

Comment:
lets lock this to a specific version number as we have a very security conscious app

Reply:
will change it to be the same as the others being used on servers deployed previously.

Reply:
changed it to match a specific version rather than :latest